README.md

```diff
@@ -3,25 +3,28 @@
 
 This is the repository containing the core components and documentation of the _tranSMART_ platform,
 an open source data sharing and analytics platform for translational biomedical research. tranSMART
-is maintained by the [tranSMART Foundation](http://transmartfoundation.org). Official releases
-can be found on the tranSMART Foundation website, and the tranSMART Foundation's development repositories
+is maintained by the [i2b2 tranSMART Foundation](http://transmartfoundation.org). Official releases
+can be found on the Foundation website, and the Foundation's development repositories
 can be found at <https://github.com/transmart/>.
 
-All the instructions on how to install, build and run a private instance of tranSMART, get set up for developing or upgrade to the latest version of tranSMART from an older version are available [in the documentation.](https://USERNAME.github.io/transmart-core/.)  For details on contributing code changes via pull requests, [see the Contributing document.](CONTRIBUTING.md)
+All the instructions on how to install, build and run a private instance of tranSMART, get set up for developing or upgrade to the latest version of tranSMART from an older version are available [in the documentation.](docs/index.md)
+
+For details on contributing code changes via pull requests, [see the Contributing document.](CONTRIBUTING.md)
 
 ## Overview
 
 The platform provides an API, which is available under [transmart-rest-api](transmart-rest-api).
 Its `v1` endpoints are documented there, the `v2` endpoints are documented using Swagger in [open-api](open-api).
 There are two main applications available: [transmart-server](transmart-server) and [transmart-api-server](transmart-api-server), both exposing the API and extended by multiple plugins.
-The OAuth2 authentication of the API for transmart-server is managed by [transmart-oauth](transmart-oauth) plugin, while transmart-api-server uses an external identity provider.
+The OAuth2 authentication of the API for transmart-server is managed by [transmart-oauth](transmart-oauth) plugin, while transmart-api-server uses an external identity provider - [Keycloak](https://www.keycloak.org/).
 
-As an user interface there is a frontend application [transmartApp](transmartApp) built as a "web" profiled Grails platform plugin and supported by transmart-server, and an Angular4 based
+As an user interface there is a frontend application [transmartApp](transmartApp) built as a "web" profiled Grails platform plugin and supported by transmart-server, and an Angular 6 based
```

Comment:
I would replace Angular <version> completely with single page application or smth like that.
IMHO information about the framework is not important at this point.